packages/cdk8s/test/yaml.test.ts

```diff
@@ -79,6 +79,15 @@ describe('save', () => {
     expect(fs.readFileSync(outputFile, 'utf-8')).toMatchSnapshot();
   });
 
+  test('strings are not folded', () => {
+    const longStringList = [
+      '^(((\d*(\.\d*)?h)|(\d*(\.\d*)?m)|(\d*(\.\d*)?s)|' +
+      '(\d*(\.\d*)?ms)|(\d*(\.\d*)?us)|(\d*(\.\d*)?µs)|' +
+      '(\d*(\.\d*)?ns))+|infinity|infinite)$',
+    ];
+    const dumpPath = Yaml.tmp(longStringList);
+    expect(fs.readFileSync(dumpPath, 'utf8').trimEnd()).not.toContain('\n');
```

Comment:
The assertion here is not that the output does not contain a newline but rather that the escaping of `\*` does not cross line boundaries.

Fix the test name to reflect.

Reply:
I see. I fixed the test name `escaped character does not divided by cross line boundries`.